tests/s25Main/UI/testSmartBitmap.cpp

```diff
@@ -216,22 +216,24 @@ BOOST_AUTO_TEST_CASE(MultiPlayerBitmap)
     glSmartBitmap smartBmp;
     for(int i = 0; i < 3; i++)
     {
-        bmps.emplace_back(createRandPlayerBmp(70));
+        bmps.emplace_back(createRandPlayerBmp(50));
         smartBmp.add(bmps.back().get());
     }
+
+    Extent size = smartBmp.GetSize();
```

Comment:
diese änderungen haben jetzt aber nicht wirklich was mit Upnp zu tun?

Reply:
Hatte hauptsächlich in den Submodules (libutil) was gemacht, aber dann sind die CI tests gefailt. Da ich dort einige Random Tests (werte sind nicht fest sondern zufällig, vgl "fuzzy testing") eingebaut habe war das fast schon zu erwarten. Kann aber nen extra MR nur mit den Testfixes machen, wenn gewünscht.